pkg/util/mount/mount.go

```diff
@@ -32,7 +37,19 @@ type Interface interface {
 	List() ([]MountPoint, error)
 	// IsMountPoint determines if a directory is a mountpoint.
 	IsMountPoint(file string) (bool, error)
+	SetRunner(executor ContainerExecutor)
```

Comment:
When Mounter instance is created (cmd/kubelet/app/server.go), Kubelet does not exist yet. It's created later and the ContainerMounter must get a reference to it to run commands in pods/containers. The point was to keep Kubelet not knowing what mounter implementation is being used, it allows for easier refactoring.